client/src/components/ElementEditor/AddNewButton.js

```diff
@@ -17,9 +17,9 @@ class AddNewButton extends Component {
   }
 
   toggle() {
-    this.setState({
-      popoverOpen: !this.state.popoverOpen
-    });
+    this.setState((prevState) => ({
```

Comment:
Do we need to make this change to get eslint to pass? I'd prefer not to make changes like this in a linting PR

Reply:
See [this comment](https://github.com/silverstripe/silverstripe-elemental/pull/1070#discussion_r1253833874)